Kitodo/src/main/java/org/kitodo/forms/TemplateForm.java

```diff
@@ -395,6 +401,94 @@ public void setTask(Task task) {
         this.task = task;
     }
 
+    /**
```

Comment:
Is it needed to generate images for templates (Produktionsvorlagen)?

Reply:
No, the user only chooses in the template what to generate and what to validate. It is then generated or validated in the task. Look again at the page “Generieren von Bildern”, in particular the points 3 and 5. For the validating it is the same as the generation, only that the pictures are not generated, but validated. Or, if the user wants that, of course, both, so that they are first generated and then validated, whether they are OK.